src/Executables/Game/Extensions/PaketExtensions.cs

```diff
@@ -13,7 +13,7 @@ public static Character ToCharacter(this PlayerData player)
             Name = player.Name,
             Class = player.PlayerClass,
             Level = player.Level,
-            Playtime = player.PlayTime,
+            Playtime = player.PlayTime == 0 ? 0 : (uint) player.PlayTime / 60000, // Milliseconds to minutes
```

Comment:
Maybe a more readable approach would be `TimeSpan.FromMiliSeconds(player.PlayTime).TotalMinutes` but it's up to you :)